src/components/wb-playground/kyn-linked-div-wb/kyn-linked-div-wb.ts

```diff
@@ -0,0 +1,87 @@
+import { html, LitElement } from 'lit';
+import Styles from './kyn-linked-div-wb.scss';
+
+import '@kyndryl-design-system/shidoka-foundation/components/link';
+import { customElement, property } from 'lit/decorators.js';
+
+////// INCLUDE IN A TARGET PROP
+
+/**
+ * Linked div.
+ */
+@customElement('kyn-linked-div-wb')
+export class KynLinkedDivWb extends LitElement {
+  static override styles = [Styles];
+
+  /**
+   * Item size, required.
+   * @type {string}
+   * @default 'auto'
+   */
+  @property({ type: String })
+  size = 'auto';
```

Comment:
You can skip the JSDoc @type and @default comments. The analyzer will automatically infer those from the code below.